pkg/metrics/metrics.go

```diff
@@ -14,6 +14,7 @@ const (
 	DefaultHealthCheckMetricsAddress = ":8083"
 	DefaultMachineSetMetricsAddress  = ":8082"
 	DefaultMachineMetricsAddress     = ":8081"
+	DefaultMetal3MetricsAddress      = ":60000"
```

Comment:
Would be good to know the motivation behind choosing this number, any reason not to try `8085`? (To match the exposed port)